src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1402,14 +1395,30 @@ void uct_dc_mlx5_iface_set_ep_failed(uct_dc_mlx5_iface_t *iface,
     ucs_status_t status;
     ucs_log_level_t log_lvl;
 
+    if (ep->flags & (UCT_DC_MLX5_EP_FLAG_NO_ERR_HANDLER |
```

Comment:
UCT_DC_MLX5_EP_FLAG_NO_ERR_HANDLER  -> UCT_DC_MLX5_EP_FLAG_ERR_HANDLER_INVOKED
(same for rc)